packages/analytics-core/src/index.js

```diff
@@ -289,7 +289,7 @@ function analytics(config = {}) {
    * @property {On} on - Fire callback on analytics lifecycle events.
    * @property {Once} once - Fire callback on analytics lifecycle events once.
    * @property {GetState} getState - Get data about user, activity, or context.
-   * @property {Storage} storage - storage methods
+   * @property {AnalyticsStorage} storage - storage methods
```

Comment:
This conflicts with the built-in type `Storage` used for `localStorage` & `sessionStorage` so I took the liberty of renaming it while I was in here to fix the TypeScript error I got.  Hopefully that's OK.